index.html

```diff
@@ -41,5 +41,19 @@
     </a>
 </div>
 </div>
-    </body>
+
+<div class="ui divider"></div>
+<div class="ui grid">
+    <div class="five wide column">
+        <a class="twitter-timeline" data-height="640" data-width="520" href="https://twitter.com/mountsion_cbs?ref_src=twsrc%5Etfw">Tweets by mountsion_cbs</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Can you change `data-width` to equal 100% as you have it set to a strict number this could potentially cause problems to the grid under different size screens, as for the `data-height` we will revisit it when more content is added to the home page